resources/js/components/groups/edit-group-modal.vue

```diff
@@ -0,0 +1,184 @@
+<template>
+  <modal
+    title="Edit existing group"
+    save-text="Save"
+    :save-function="save"
+    :close-function="close"
+    ref="modal"
+  >
+    <label class="label">Group name</label>
+    <input
+      type="text"
+      placeholder="Where are we gonna eat ?"
+      class="input input-group-name"
+      v-model="group.name"
+    />
+    <div class="columns">
+      <div class="column is-half">
+        <select-list
+          icon="fa-user-plus"
+          placeholder="Member email"
+          :items="displayedUsers"
+          :add-function="addUser"
+          :remove-function="removeUser"
+          ref="userList"
+        >Members</select-list>
+      </div>
+      <div class="column is-half">
+        <select-list
+          icon="fa-plus"
+          placeholder="Item name"
+          :items="displayedItems"
+          :add-function="addItem"
+          :remove-function="removeItem"
+        >Default items</select-list>
+      </div>
+    </div>
+  </modal>
+</template>
+
+<script>
+import modal from "../modal";
+import selectList from "../groups/select-list";
+
+export default {
+  components: {
+    modal,
+    selectList
+  },
+  data() {
+    return {
+      group: {},
+      usersToAdd: [],
+      usersToRemove: [],
+      itemsToAdd: [],
+      itemsToRemove: [],
+      displayedUsers: [],
+      displayedItems: []
+    };
+  },
+  methods: {
+    open: function(id) {
+      this.axios
+        .get("/groups/" + this.group.id)
+        .then(response => {
+          this.group = response.data.data;
+
+          let ownId = localStorage.getItem("id");
+          this.displayedUsers = this.group.users.filter(
+            user => user.id !== ownId
+          );
+
+          this.$refs.userList.unremovableItems = this.group.users.filter(
+            user => user.id === ownId
+          );
+          this.$refs.userList.unremovableItems[0].name += " (You)";
+          this.$refs.modal.active = true;
+        })
+        .catch(error => {
+          alert("ERROR: Could not load group");
+        });
+    },
+    save: function() {
+      this.axios.patch("/groups/" + this.group.id, {
+        name: this.group.name,
+        users_to_add: this.usersToAdd.map(user => user.id),
```

Comment:
users_to_add should receive
`[{"id": <id>, "admin": <bool>}]`
users_to_remove and items_to_remove should receive
`[{"id": <id>}]`